README.md

```diff
@@ -3,6 +3,7 @@ This is a repo to practice pull request, and contributing to projects, on Github
 You have just to fork this repo to add your name, and maybe link to your GitHub account.
 
 * Mahfoudh Arous [@aladin002dz](https://github.com/aladin002dz)
+* BARAKA LARBI [@larbibaraka](https://github.com/larbibaraka)
```

Comment:
thanks for your awesome PR, but just to remind you, it is not best practice to send PR from master branch. it is better to create a branch for the changes, and keep the master branch to update the local repo with the command "git pull upstream".